examples/dt/adoption/rhv/README.md

```diff
@@ -0,0 +1,61 @@
+# RHEV base adoption Topology
+
+This topology contains Director deployed Source OpenStack on RHEV that is networked together with RHEV deployed openshift cluster. This topology executes adoption test suite that verifies adoption from existing 17.1 osp deployment into podified openstack. The upgrade/migration to the podified OpenStack requires planning various aspectsof the environment such as node roles, planning your network topology.
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 22.12.23         |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
```

Comment:
nit: Could we change this `OpenShift Compact` instead?

There are three types `SNO | Compact | HA`

Reply:
So, it should be HA or compact?

Reply:
This would be `compact` as we are setting count to 3 meaning, controller + worker roles are co-located.

Reply:
Done